homeassistant/components/sensor/__init__.py

```diff
@@ -88,6 +88,14 @@
 class SensorDeviceClass(StrEnum):
     """Device class for sensors."""
 
+    ACCUMULATED_PRECIPITATION = "accumulated_precipitation"
```

Comment:
Should this use `homeassistant.const.ACCUMULATED_PRECIPITATION`?

Reply:
No.
All implementations of `StrEnum` currently use their own values in Home Assistant.
That would need to be a separate discussion.